starport/templates/testutil/register.go

```diff
@@ -25,5 +30,12 @@ func Register(ctx *plush.Context, gen *genny.Generator) error {
 	if !ctx.Has(modulePathKey) {
 		return fmt.Errorf("ctx is missing value for the ket %s", modulePathKey)
 	}
+	path, err := filepath.Abs(testUtilDir)
+	if err != nil {
+		return err
+	}
+	if _, err := os.Stat(path); !os.IsNotExist(err) {
+		return nil
```

Comment:
Do we want to return `nil` if the testutil dir exist?

We should return `nil` if `err == nil` and `err` if `err != nil && !os.IsNotExist(err)`

Reply:
if we return an error we break the command when we try to scaffold a second list into the module, because its tries to unbox the testutil folder again